rush-plugins/rush-azure-storage-build-cache-plugin/src/AzureStorageBuildCacheProvider.ts

```diff
@@ -3,11 +3,14 @@
 
 import { ITerminal } from '@rushstack/node-core-library';
 import { PrintUtilities } from '@rushstack/terminal';
-import type {
```

Comment:
It's still better to use type-only imports wherever possible, and to avoid dependencies on types that declare private members.